pset/blinder.go

```diff
@@ -1,23 +1,25 @@
 package pset
 
 import (
-	"crypto/rand"
 	"errors"
 	"github.com/btcsuite/btcd/btcec"
 	"github.com/vulpemventures/go-elements/confidential"
 	"github.com/vulpemventures/go-elements/transaction"
 	"github.com/vulpemventures/go-secp256k1-zkp"
 )
 
+type randomNumberGenerator func(n int) ([]byte, error)
+
 type blinder struct {
 	pset             *Pset
 	blindingPrivkeys [][]byte
 	blindingPubkeys  [][]byte
+	rng              randomNumberGenerator
 }
 
 // NewBlinder returns a new instance of blinder, if the passed Pset struct is
 // in a valid form, else an error.
-func NewBlinder(pset *Pset, blindingPrivkeys, blindingPubkeys [][]byte) (
+func NewBlinder(pset *Pset, blindingPrivkeys, blindingPubkeys [][]byte, rng func(n int) ([]byte, error)) (
```

Comment:
use the defined type:
```suggestion
func NewBlinder(pset *Pset, blindingPrivkeys, blindingPubkeys [][]byte, rng randomNumberGenerator) (
```